NMWeb/e2e-testcafe/testSrc/profile/userProfile.testc.ts

```diff
@@ -9,9 +9,6 @@ const visitGeoTag = Selector("app-user-geo-location[formControlName='whereILive'
 const twitterInput = Selector('app-other-profile-user-name input');
 
 const closeTagButton = Selector('.ion-close-circled');
-const acceptGeoLocationButton = Selector(
-  'body > app-root > div > mat-sidenav-container > mat-sidenav-content > div > app-user-profile-details > form > app-user-geo-locations > mat-card > mat-card-content > div > app-user-geo-location:nth-child(1) > app-user-pick-location > p-dialog > div > div.ui-dialog-content.ui-widget-content > button'
```

Comment:
This was always selecting the first child which corresponded to the first popup (where I live). Now it is relative to the selected popup